### ISSUE TITLE
Segmentation fault in computeWellTransmissibility

### ISSUE THREAD

**Issue by philippechehade** [GEOSX/GEOSX#2263]
I have a model successfully initialized without wells.

When I tried adding an injection well to the model, the below error was obtained: 
```
MainGrid/Level0/wellregion/wellregionUniqueSubRegion/RELPERM allocated 1 quadrature points
Received signal 11: Segmentation fault ** StackTrace of 11 frames **
Frame 0: /lib64/libc.so.6 
Frame 1: geosx::PerforationData::computeWellTransmissibility(geosx::MeshLevel const&, geosx::WellElementSubRegion const&, LvArray::Array<LvArray::Array<LvArray::ArrayView<double const, 3, 2, int, LvArray::ChaiBuffer>, 1, camp::int_seq<long, 0l>, int, LvArray::ChaiBuffer>, 1, camp::int_seq<long, 0l>, int, LvArray::ChaiBuffer> const&) 
Frame 2: geosx::CoupledReservoirAndWellsBase<geosx::CompositionalMultiphaseBase, geosx::CompositionalMultiphaseWell>::initializePostInitialConditionsPreSubGroups()::{lambda(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, geosx::MeshLevel&, LvArray::ArrayView<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, 1, 0, int, LvArray::ChaiBuffer> const&)#1}::operator()(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, geosx::MeshLevel&, LvArray::ArrayView<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, 1, 0, int, LvArray::ChaiBuffer> const&) const::{lambda(int, geosx::WellElementSubRegion&)#1}::operator()(int, geosx::WellElementSubRegion) const 
Frame 3: void geosx::SolverBase::forDiscretizationOnMeshTargets<geosx::CoupledReservoirAndWellsBase<geosx::CompositionalMultiphaseBase, geosx::CompositionalMultiphaseWell>::initializePostInitialConditionsPreSubGroups()::{lambda(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, geosx::MeshLevel&, LvArray::ArrayView<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, 1, 0, int, LvArray::ChaiBuffer> const&)#1}>(geosx::dataRepository::Group&, geosx::CoupledReservoirAndWellsBase<geosx::CompositionalMultiphaseBase, geosx::CompositionalMultiphaseWell>::initializePostInitialConditionsPreSubGroups()::{lambda(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, geosx::MeshLevel&, LvArray::ArrayView<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, 1, 0, int, LvArray::ChaiBuffer> const&)#1}&&) const 
Frame 4: geosx::CompositionalMultiphaseReservoirAndWells<geosx::CompositionalMultiphaseBase>::initializePostInitialConditionsPreSubGroups() 
Frame 5: geosx::dataRepository::Group::initializePostInitialConditions() 
Frame 6: geosx::dataRepository::Group::initializePostInitialConditions() 
Frame 7: geosx::dataRepository::Group::initializePostInitialConditions() 
Frame 8: geosx::GeosxState::applyInitialConditions() 
Frame 9: main 
Frame 10: __libc_start_main 
Frame 11: /workrd/SCR/GEOSX/install/gcc8/GEOSX-adc36ad/bin/geosx 
=====
```

As a user, I am stuck in front of this error and I have no idea on what to correct. Is the problem coming from the well's coordinates? from the well's definition? from the XML ? I have no clear information on my error. 

@jeannepellerin

**Comment by dkachuma** [comment]
I can reproduce the error with the attached small case [voidage.xml.gz](https://github.com/GEOSX/GEOSX/files/10564078/voidage.xml.gz).

The problem is that the well perforation is in a region that is not targeted by the fluid solver. Removing the `CellElementRegion` highlights the actual problem more.

**Comment by francoishamon** [comment]
Closed by #2269
